pkg/monitor/lease.go

```diff
@@ -86,16 +110,139 @@ func leaseVIP(cfgPath, masterDevice, name string) error {
 		return err
 	}
 
+	lease_file := getLeaseFile(cfgPath, name)
+	if f, err := os.Create(lease_file); err != nil {
+		log.WithFields(logrus.Fields{
+			"name": lease_file,
+		}).WithError(err).Error("Failed to create lease file")
+		return err
+	} else {
+		f.Close()
+	}
+
+	if err := WatchLeaseFile(log, lease_file, iface.Name, ip); err != nil {
+		log.WithFields(logrus.Fields{
+			"filename":      lease_file,
+			"expectedIface": iface.Name,
+			"expectedIp":    ip,
+		}).WithError(err).Error("Failed to watch the lease file")
+		return err
+	}
+
 	cmd := exec.Command("dhclient", "-d", "--no-pid", "-sf", "/bin/true",
-		"-lf", getLeaseFile(cfgPath, name), "-v", iface.Name, "-H", name)
+		"-lf", lease_file, "-v", iface.Name, "-H", name)
```

Comment:
We are watching the file, but we will not know if the dhclient terminated for some reason.  Shouldn't we monitor the process as well and maybe restart if it terminates?